holoviews/core/data/dictionary.py

```diff
@@ -101,7 +101,10 @@ def init(cls, eltype, data, kdims, vdims):
 
     @classmethod
     def validate(cls, dataset):
```

Comment:
Why not make the ``derived_vdims`` flag (or similar) and explicit argument to ``validate``?

Reply:
Now removed.